docs/advanced-usage/compute-backends/kubernetes/index.md

```diff
@@ -17,3 +17,66 @@ on the default backend.
         commands:
             - python "${helloworld}"
 ```
+
+## Custom memory limit
+
+Kubernetes allows configuring and managing resources for containers. With REANA, you can configure cluster wide **memory** limits to make the effiecient use of available resources.
+
+To set the memory limit of a job you must specify `kubernetes_memory_limit`.
+
+This can be set directly in `bytes` and you can read more about the expected memory values on [kubernetes official documentation](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory)
```

Comment:
>  This can be set directly in `bytes`

Although this is possible, I don't think it's the most common approach as it's not human-readable. The user can also specify human-readable units, so I'd shorten this part.